### PR TITLE
Allow the setting of candy threshold

### LINKED ISSUE #4256: Ignore Pokemon based on candy
### Short Description

Ignore Pokemon based on the number of candies

### Possible solution

    candy = inventory.candies(True).get(pokemon.pokemon_id)
    if candy > candy_threshold:
        ignore()
    else:
        catch()

in config.json:

    "catch": {
          "any": {
                "candy_threshold": 200
          }
     },

### How it would help others

It will benefit collection-only users. Keep account exp/h low.

### PR DESCRIPTION
**Pull request by bbiiggppiigg**
## Short Description:
Allow the setting of candy threshold to ignore certain pokemon if the threshold is reached
Using a default value of 400 ( For Magikarp )

## Fixes/Resolves/Closes (please use correct syntax):

Closes #4256 

-
-
-

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (k4n30) [inline .pullapprove.yml]: Why the change?

Comment (bbiiggppiigg) [inline .pullapprove.yml]: I'm new to pullapprove and accidentally changed the file.
I'll redo the pull request
